### PR TITLE
Place editor, copy and paste of lat and long text

### PR DESCRIPTION
**Pull request by SNoiraud**
no longer auto-populating latitude and longitude fields

Fixes #012349

### CODE REVIEW

File: gramps/gui/editors/editplace.py [hunk 1]
```diff
@@ -187,9 +187,14 @@ def _setup_fields(self):
...
     def set_latlongitude(self, value):
         try:
-            coma = value.index(', ')
-            longitude = value[coma+2:].strip().replace(',','.')
-            latitude = value[:coma].strip().replace(',','.')
+            # Bug 12349
+            parts = value.split(', ')
+            if len(parts) == 2:
+                longitude = parts[0].strip().replace(',', '.')
+                latitude = parts[1].strip().replace(',', '.')
```
Comment (zertrin) [inline gramps/gui/editors/editplace.py:194]: This is reversed! Latitude is the first field and longitude the second one according to the label above the text input.

See the result for a village in Belgium:

image

Comment (emyoulation) [reply gramps/gui/editors/editplace.py:194]: A patch has been posted. 

See 
https://gramps.discourse.group/t/5-1-4-bug-in-gps-location-dialog/1731

Comment (zertrin) [reply gramps/gui/editors/editplace.py:194]: Yes only saw the existing patch #1290 after submitting mine #1320 😄 

Sorry for the noise.

Comment (emyoulation) [reply gramps/gui/editors/editplace.py:194]: It is noise in the right place if it helps the next person looking here.

**Pull request merged**